EIPS/eip-7201.md

```diff
@@ -4,7 +4,8 @@ title: Namespaced Storage Layout
 description: A formula for the storage location of structs in the namespaced storage pattern.
 author: Francisco Giordano (@frangio), Hadrien Croubois (@Amxx), Ernesto García (@ernestognw), Eric Lau (@ericglau)
 discussions-to: https://ethereum-magicians.org/t/eip-7201-namespaced-storage-layout/14796
-status: Draft
+status: Last Call
+last-call-deadline: 2023-09-15
```

Comment:
```suggestion
last-call-deadline: 2023-10-7
```

Reply:
We've actively requested and received feedback for a while already as seen in the [discussions thread](https://ethereum-magicians.org/t/eip-7201-namespaced-storage-layout/14796).

We're going to release code using this EIP at the beginning of October so it will become de facto Final at that point, since as authors we're not going to accept normative changes after the code is released. I'll set the last call deadline to Oct 1st to match this reality.